pwndbg/lib/disasm/helpers.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+
+def to_signed(unsigned: int, bit_width: int):
+    """
+    Returns the signed number associated with the two's-complement binary representation of `unsigned`
+    """
+    extract_bit = 1 << (bit_width - 1)
+    return unsigned - ((unsigned & extract_bit) << 1)
```

Comment:
Just a note: only the `to_signed` function is used from here for now. I wonder if we really need other functions, but I guess/hope we do.